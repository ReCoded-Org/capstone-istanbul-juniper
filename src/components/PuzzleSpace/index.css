.puzzlePage__gameSpace {
  background-color: #b4d0c4;
  height: 100vh; /* use of vh is temporary until the game content is ready */
  display: flex;
  justify-content: center;
  align-items: center;
}

.puzzlePage__puzzleBox {
  background-color: #3b8c79;
  width: 40%;
  height: 50%;
  display: flex;
  flex-direction: column;
  justify-content: stretch;
}
