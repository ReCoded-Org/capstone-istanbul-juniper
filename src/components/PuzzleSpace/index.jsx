import React from "react";
import CurrentImage from "../CurrentImage/index";
import ListOfImages from "../ListOfImages/index";
import { Layout, Card } from "antd";
import "./index.css";

const GameSpace = () => {
  const { Content } = Layout;
  return (
    <Content className="puzzlePage__gameSpace">
      <Card size="small" className="puzzlePage__puzzleBox">
        <ListOfImages />
        <CurrentImage />
      </Card>
    </Content>
  );
};

export default GameSpace;
