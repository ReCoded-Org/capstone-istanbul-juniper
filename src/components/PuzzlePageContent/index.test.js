import React from "react";
import PuzzleSpace from "./index.jsx";
import renderer from "react-test-renderer";

describe("PuzzleSpace Component", () => {
  it("matches the tree", () => {
    const tree = renderer.create(<PuzzleSpace />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
