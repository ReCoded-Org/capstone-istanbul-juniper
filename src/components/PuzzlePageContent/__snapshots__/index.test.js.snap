// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PuzzleSpace Component matches the tree 1`] = `
<main
  className="ant-layout-content puzzlePage__content"
>
  <div
    className="ant-card ant-card-bordered ant-card-small puzzlePage__content-puzzleBox"
  >
    <div
      className="ant-card-body"
      style={Object {}}
    />
  </div>
</main>
`;
